lua/impatient/cachepack.lua

```diff
@@ -1,16 +1,26 @@
 local ffi = require('ffi')
 
 -- using double for packing/unpacking numbers has no conversion overhead
-local sizeof_c_double = ffi.sizeof("double")
-local c_double = ffi.typeof("double[1]")
+if vim.loop.os_uname().machine ~= "x86_64" then
```

Comment:
Don't forget about arm64 which accounts for everyone using a modern Mac.

Reply:
It may be better to use `jit.os`

Reply:
I was discussing it with @chase and he suggested we only use the `int` type on `jit.arch=='arm'`. What do you think?

Reply:
Sure.